Composer/packages/adaptive-form/src/components/FormRow.tsx

```diff
@@ -35,13 +33,7 @@ export const getRowProps = (rowProps: FormRowProps, field: string) => {
   const { required = [] } = schema;
   const fieldSchema = resolvePropSchema(schema, field, definitions);
 
-  const intellisenseScopes: string[] = [];
-  if (field === 'property') {
-    intellisenseScopes.push('variable-scopes');
```

Comment:
I see the logic is deleted from FormRow, but where is it added in IntellisenseField?  I only see the use of projectId in IntellisenseField.tsx.

Reply:
Hi @GeoffCoxMSFT, I made sure that Intellisense can consume the data from the uiSchema in the 1st Intellisense PR. 
This is what `uiOptions.properties?.[field] `does.
I only needed the hardcoding as a temporary thing until the data was in the schema.

Reply:
One thing to be aware of is that users with old ui schemas will not get this behavior. Maybe we keep the hardcoded path for now, but prefer the UI Schema options if they are set?

Reply:
Is the uiSchema not bundled with the code? Aka won't users with the latest app version not also have the latest uiSchema? I would rather avoid any hardcoding when we can. Any field with the "property" value will get one type of Intellisense, which might lead to unexpected behaviors if that value is used for a different purpose (aka not a variable setter) somewhere in the schema.